js-core/homeworks/homework-6/src/main.js

```diff
@@ -0,0 +1,292 @@
+ /** TASK 1
+ * Посчитайте количество букв а, в передаваемом параметре
+ *
+ * */
+
+let randomString = 'aaa b a w c ';
+
+let user = {
+  name: 'Albina',
+};
+
+let javaScript = {
+  html: 'JavaScript',
+};
+
+function countLetterA(str) {
+	let counter = 0;
+	let arr = str.toLowerCase().split('');
+	console.log(arr);
+	arr.forEach((item) => {
+		if (item == 'a') {
+			counter++;
+		};
+	})
+
+	return counter;
+}
+
+console.log('task 1-1 ---->', countLetterA(randomString)); // 4
+console.log('task 1-2 ---->', countLetterA(user.name + javaScript.html)); // 3 //4
+//------------------------------------------------------------------
+
+/*
+ *
+ * TASK 2
+ * Вам дано предложение, вам необходимо перевернуть каждое
+ * слово в предложении в обратном порядке
+ *
+ * */
+//1. Разбить предложение на массив по пробелу
+//2. взять каждый элемент массива и тоже разбить на массив по ''
+//3. Применить метод реверс на каждом слове
+//4. Собрать получившийся массив опять в строку
+function reverseEachWord(str) {
+	let strArr = str.split(' ');
+	let word = strArr.map((item) => {
```

Comment:
aha, arrow function, okay :) But we wouldn't use it for some time in class

Reply:
that syntax with 'function' confused me a bit, because I used array methods few times and don't remember a word "function" ))